test/multest.jl

```diff
@@ -31,3 +31,9 @@ qv = QuArray(v)
 
 # a simple test of the `==` operator
 @assert qm*3im == scale!(3im, copy(qm))
+
+# tests for Matrix Division
+v1 =  [0.5+0*im, 0.+0.*im]
+qv1 = normalize!(QuArray(v1))
+@assert coeffs(\(sigmaz, sigmax)) == sparse([0. 1.;-1. 0.])
```

Comment:
Try to use `full(sigmaz)` and `full(sigmax)` here and in the next line. As I said via PM it might actually be better for the Pauli matrices to be dense anyways, but this can be done separately.